### PR TITLE
Feature/initialize

### PR DESCRIPTION
**Pull request by AmineDerbal**
in this milestone ♻️ 

- setup a new react application 👍 
- create our reusable Components 👍 
- setup 2 routes in our app 👍 
- the route "/" is the Home route containing a list of books and a form to add books 👍 
- the route "Categories" contain a button that checks status 👍 
- No logic or styling was implemented for now 👍

### CODE REVIEW

File: src/components/BookLists.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  return (
+    <div>
+      {books.map((book) => (
+        <p key={book.id}>
+          <strong>{book.title}</strong>
+          {' '}
+          <span>by</span>
+          {' '}
+          <strong>{book.author}</strong>
+        </p>
```
Comment (gemmen29) [inline src/components/BookLists.js]: - [ ] Kindly create a separate component called Book for displaying a single book and reuse it here as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v2_0.md#:~:text=Create%20a%20component%20called%20Book%20for%20displaying%20a%20single%20book%20(receive%20a%20title%20and%20an%20author%20as%20prop)%20and%20reuse%20it%20in%20a%20component%20that%20displays%20a%20list%20of%20books.).

File: src/components/BookLists.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+  return (
+    <div>
+      {books.map((book) => (
+        <p key={book.id}>
+          <strong>{book.title}</strong>
+          {' '}
+          <span>by</span>
+          {' '}
+          <strong>{book.author}</strong>
```
Comment (gemmen29) [inline src/components/BookLists.js]: - [ ] Kindly add a remove button for each book as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v2_0.md#:~:text=with%20the%20Remove%20button). There is no functionality for delete needed yet.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @AmineDerbal, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookLists.js (hunk 1 above)
Comment (AmineDerbal) [reply src/components/BookLists.js]: This is the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v3_0.md) microverse give us in our progress page

File: src/components/BookLists.js (hunk 2 above)
Comment (AmineDerbal) [reply src/components/BookLists.js]: This is the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v3_0.md) microverse give us in our progress page

**Comment by AmineDerbal** [comment]
- add book component 👍 
- add a delete button to the Book component 👍 
- display the list of books from the Book component 👍

**Review by m05e5 — APPROVED** [review]
**Hi @AmineDerbal 
You did great. Your project is complete!  There is nothing else to say other than... it's time to merge:shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights

✔️ Professional readme file
✔️ Clean Codebase

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Comment by AmineDerbal** [comment]
Thank you

**Review by nicupop729 — COMMENTED** [review]
Hi @AmineDerbal,

We apologise for the misunderstanding. 
The project requirements were just updated, and the first reviewer had access to the old file. We updated the material after taking note of the circumstance so that the reviewers could also access the new, updated file.

Note for the Quality Control checks: @gemmen29 reviewed under the old Project Requirements file and his review was genuine.

**Comment by AmineDerbal** [comment]
@nicupop729 
Understood and thanks for your clarification, and I also apologize to @gemmen29 if I was harsh on him

**Pull request merged**